metrics/web/docs/features/builds_aggregation/diagrams/build_days_domain_layer_class_diagram.puml

```diff
@@ -0,0 +1,47 @@
+@startuml build_days_domain_layer_class_diagram
+
+package base.domain.usecases {
+    interface UseCase<Type, Params> {
+        call: Type
+    }
+}
+
+package common.domain {
+    package entities {
```

Comment:
It seems like entities in this package are closely related to the `dashboard` module, so I believe we should think about placing them under the `dashboard` package. The same for the `ReceiveBuildDayProjectMetricsUpdates` use case. What do you think?